docs/source/examples/index.rst

```diff
@@ -50,6 +50,7 @@ The notebooks are organised into subfolders, and can be viewed in the galleries
     notebooks/models/compare-particle-diffusion-models.ipynb
     notebooks/models/composite_particle.ipynb
     notebooks/models/coupled-degradation.ipynb
+    notebooks/models/DCHS.ipynb
```

Comment:
this is an uncommon acronym, can you write out the full name?

Reply:
Sure, I was calling it differential capacity hysteresis state. I will write this out in full for the notebook